aspnetcore/security/app-secrets/samples/1.x/UserSecrets/Startup3.cs

```diff
@@ -0,0 +1,40 @@
+﻿#if never
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using UserSecrets.Models;
+
+namespace UserSecrets
+{
+    public class Startup
+    {
+        private string _moviesApiKey = null;
+
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            #region snippet_BindToObjectGraph
+            var moviesConfig = new MovieSettings();
+            Configuration.GetSection("Movies").Bind(moviesConfig);
```

Comment:
I noticed that you used the fancier binding syntax for the 2.x version ...

```csharp
var moviesConfig = Configuration.GetSection("Movies").Get<MovieSettings>();
```

... and I thought that by-convention that we were documenting (in samples) the latest patch release. Since that came out in 1.1, should this also use the sweeter approach?

Reply:
The intent here was to show folks on 1.0 what to do. The newer `Get<T>` approach doesn't work there.

Reply:
> It wasn't worth the time to create a 1.0 app

You mean "1.1" app?

That's cool. I didn't know that the app was still 1.0.

Reply:
There's a 1.1 and a 2.1 sample app. The 1.1 app uses the older approach. I have a moniker range for 1.0 in the doc that displays the older approach.